### PR TITLE
docs: Update doc to add commonLabels and commonAnnotations

### PR DESCRIPTION
**Pull request by Viktorsubota**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

Updating docs according to the changes made in [this PR](https://github.com/Flagsmith/flagsmith-charts/pull/162)

## How did you test this code?

No tests, as it's just a docs update. The change itself was tested, see its PR.

### CODE REVIEW

File: docs/docs/deployment/hosting/kubernetes.md [hunk 1]
```diff
@@ -455,6 +455,8 @@ The following table lists the configurable parameters of the chart and their def
 | `api.statsd.hostFromNodeIp`                        | Set as true to use the node IP as the statsd host instead        | `false`                        |
 | `api.statsd.port`                                  | Host port to receive statsd metrics                              | `8125`                         |
 | `api.statsd.prefix`                                | Prefix to add to metric ids                                      | `flagsmith.api`                |
+| `commonLanesls`                                    | Labels to add to all resources                                   | `{}`                           |
```
Comment (dabeeeenster) [inline docs/docs/deployment/hosting/kubernetes.md]:
```suggestion
| `commonLabels`                                    | Labels to add to all resources                                   | `{}`                           |
```

This looks like a typo?

**Comment by dabeeeenster** [comment]
Can you check the table formatting with the precommit linter?

**Comment by Viktorsubota** [comment]
> Can you check the table formatting with the precommit linter?

Sure, the pre-commit failed, but I think, it's unrelated.
image

**Review by matthewelwell — APPROVED** [review]
This looks good to me, I've added a comment to the PR on the charts repo with a question but this can be merged once we're happy with the PR on the charts repo. Thanks @Viktorsubota !

**Comment by Viktorsubota** [comment]
> This looks good to me, I've added a comment to the PR on the charts repo with a question but this can be merged once we're happy with the PR on the charts repo. Thanks @Viktorsubota !

Hey @matthewelwell,
Sorry, I don't see any comments there.  

Could you point me to the one you left? 
Thanks!

**Comment by matthewelwell** [comment]
Hi @Viktorsubota, I think at the time I had forgotten to actually submit the review but I have done that now. Could you check the comment I left [there](https://github.com/Flagsmith/flagsmith-charts/pull/162)?

**Comment by Viktorsubota** [comment]
@dabeeeenster, @matthewelwell, Hey guys, [the charts PR](https://github.com/Flagsmith/flagsmith-charts/pull/162) got merged, can we merge this PR as well?

**Pull request merged**